OpenCombine.py

```diff
@@ -0,0 +1,21 @@
+# To use `OpenCombine.py`, figure out its full path.
+# Let's say the full path is `~/projects/OpenCombine/OpenCombine.py`.
+# Then add the following statement to your `~/.lldbinit` file:
+#
+#     command script import ~/projects/OpenCombine/OpenCombine.py
+
+import lldb
+
+# Show a Demand as either `max(N)` or `unlimited`.
+def summary_Demand(sb_value, internal_dict):
+    child = sb_value.GetChildAtIndex(0)
+    if not child.IsValid():
+        return 'failed to get child of Demand'
+    number = child.GetValueAsUnsigned()
+    if number == 9223372036854775808:
```

Comment:
This number would be different on 32 bit platforms, is it possible to handle that?

Reply:
Good catch. I'll have to do a little digging to figure out the LLDB API that reports the target's word size.